### PR TITLE
adds extra links to where the classroom of Founders and Coders of Nazareth is located

### PR DESCRIPTION
**Pull request by HilbertSpitzer**
relates https://github.com/foundersandcoders/nazareth-programme/issues/161

### CODE REVIEW

File: _harp/apply/_nazareth.md [hunk 1]
```diff
@@ -6,18 +6,14 @@ We plan to conduct two rounds of interviews for the October course, the first in
 
 <h2 align='center'>CANDIDATES APPLYING FROM ISRAEL</h2>
 
-On July 20, we will invite local candidates for interviews which will be held in Nazareth on July 26 and 27.
```
Comment (shiryz) [inline _harp/apply/_nazareth.md]: Can this be replaced with the first set of interviews have been conducted?

File: _harp/apply/_nazareth.md [hunk 2]
```diff
@@ -6,18 +6,14 @@ We plan to conduct two rounds of interviews for the October course, the first in
 
 <h2 align='center'>CANDIDATES APPLYING FROM ISRAEL</h2>
 
-On July 20, we will invite local candidates for interviews which will be held in Nazareth on July 26 and 27. 
-
 We will then select up to 12 local candidates for an eight-week prep-course, where we will provide structured support for completing the [course prerequisites](./prerequisites). All those who attend all sessions of the prep-course _and_ successfully complete our course prerequisites before the end of September are guaranteed a place on the October course.
 
 Candidates who submit applications between July 20 and August 20 may be invited to interview for any remaining places on the October programme during the last week in August.
```
Comment (shiryz) [inline _harp/apply/_nazareth.md]: The first comment would be followed by this.

File: _harp/apply/_nazareth.md [hunk 3]
```diff
@@ -6,18 +6,14 @@ We plan to conduct two rounds of interviews for the October course, the first in
...
 
 <h2 align='center'>CANDIDATES APPLYING FROM OUTSIDE OF ISRAEL</h2>
 
-On July 20, we will invite international candidates who have completed the course prerequisites for interviews over Hangouts which will be held on July 26 and 27.
```
Comment (shiryz) [inline _harp/apply/_nazareth.md]: See comment above :arrow_up:

File: _harp/apply/_nazareth.md (hunk 1 above)
Comment (rebeccaradding) [inline _harp/apply/_nazareth.md]: @shiryz sure, but in a separate PR?

**Comment by rebeccaradding** [comment]
@HilbertSpitzer the link to 6170 street returns an error...

**Comment by HilbertSpitzer** [comment]
@rebeccaradding In my laptop it worked, but it could be that for others I need to add "https://" at the start to prevent these problems. [Try](https://github.com/foundersandcoders/our-site/blob/1d8aa97dceae7356f6d72c498637b4c8e302f21e/_harp/apply/_nazareth.md) clicking on the [link](https://goo.gl/ZPP82Q) again please, does the link work now?

File: _harp/apply/_nazareth.md (hunk 1 above)
Comment (shiryz) [inline _harp/apply/_nazareth.md]: Yeah but it's already been deleted

**Pull request merged**